standards/HEALTH_CHECK_SPECIFICATION.md

```diff
@@ -101,8 +115,8 @@ Field          | Type     | Required | Description
 `status`       | `string` | Yes      | The [status of the check](#check-statuses)
 `status_code`  | `int`    | No       | The status code returned by the external service (only for use with external http checks)
 `message`      | `string` | Yes      | Brief description of the status (i.e. `OK` or `received status code 500`)
-`last_checked` | `ISO8601`<sup>1</sup> | Yes | The last time the check was run
-`last_success` | `ISO8601`<sup>1</sup> | Yes | The time of the last successful check
-`last_failure` | `ISO8601`<sup>1</sup> | Yes | The time of the last failed check
+`last_checked` | `ISO8601`<sup>1</sup> | Yes | The last time the check was run (return `null` if the check has not run)
```

Comment:
Won't this always have a value?

If the check failed then there might not be a `last_success` value and if it succeeded it might not have a `last_failure`

Reply:
Another way to explain it is that if check hasn't run there won't be a check object and hence no `last_checked` field